src/fhir-tool-conversion/FhirConverter.cs

```diff
@@ -88,17 +90,26 @@ public static bool CanConvertBetweenVersions(FhirVersion from, FhirVersion to)
             where TFrom : Base
             where TTo : Base
         {
-            /*
-            string typeNameFrom = typeof(TFrom).Name;
-            var targetType = typeof(TTo);
-            string typeNameTo = targetType.Name;
-            if (typeNameFrom != typeNameTo) throw new ArgumentException($"Cannot convert from type '{typeNameFrom}' to  '{typeNameTo}'.");
+            try
+            {
+                Path = new FhirPath();
+                Path.Push(fromObject.GetType().Name);
+                return ConvertObject<TTo, TFrom>(fromObject);
+            }
+            finally
+            {
+                Path = null;
+            }
+        }
 
-             */
+        internal TTo ConvertObject<TTo,TFrom>(TFrom fromObject)
```

Comment:
To be in line with FHIR Terminology this should probably be called ConvertElement.